lib/bundles/inspec-compliance/cli.rb

```diff
@@ -198,6 +228,29 @@ def logout
 
     private
 
+    def login_automate_config(url, user, dctoken, usertoken, ent)
+      config = Compliance::Configuration.new
+      config['server'] = url
```

Comment:
we should have a `config['server_type']` to indicate the type of server. At the moment that is `compliance` and `automate`